Neos.Flow/Tests/Unit/Validation/ValidatorResolverTest.php

```diff
@@ -507,10 +530,10 @@ public function buildBaseValidatorConjunctionSkipsPropertiesAnnotatedWithIgnoreV
 
         $mockReflectionService = $this->createMock(ReflectionService::class);
         $mockReflectionService->expects(self::any())->method('getAllImplementationClassNamesForInterface')->will(self::returnValue([]));
-        $mockReflectionService->expects(self::at(0))->method('getClassSchema')->will(self::returnValue(null));
-        $mockReflectionService->expects(self::at(1))->method('getClassPropertyNames')->will(self::returnValue(['entityProperty']));
-        $mockReflectionService->expects(self::at(2))->method('getPropertyTagsValues')->with($modelClassName, 'entityProperty')->will(self::returnValue(['var' => ['ToBeIgnored']]));
-        $mockReflectionService->expects(self::at(3))->method('isPropertyAnnotatedWith')->with($modelClassName, 'entityProperty', Annotations\IgnoreValidation::class)->will(self::returnValue(true));
+        $mockReflectionService->expects(self::any(0))->method('getClassSchema')->will(self::returnValue(null));
```

Comment:
```suggestion
```
Any can be removed and the mocked method returns null anyway.

Reply:
Check.